src/dsp_tools/resources/validate_data/api-shapes.ttl

```diff
@@ -20,6 +20,12 @@
 # RDFS:LABEL SHAPE
 ########################
 
+api-shapes:rdfsLabel_Shape
```

Comment:
`sh:ignoredProperties` does not work with `dash`, which means that we have to explicitly mention the label now, which we planned on doing anyway.